doc/reference/reference_lua/pub-sub.rst

```diff
@@ -0,0 +1,172 @@
+It means that the user cannot call
+:doc:`box.broadcast('box.id', any_data) </reference/reference_lua/box_watchers/broadcast>`.
+
+Below is a table of all events that includes the name of the event and its description.
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 50 50
+
+        *   -   Built-in event
+            -   Description
+
+        *   -   :doc:`box.id </reference/reference_lua/box_info>`
+            -   Identification of the instance. Changes are particularly rare. Some
+                values never change or change only once. For example, the UUID of the instance never
+                changes after the first :doc:`box.cfg </reference/reference_lua/box_cfg>`.
+                However, it is not known before ``box.cfg`` is called.
+                The replicaset UUID is unknown until the instance joins a replicaset or
+                boots a new one, but events are supposed to start working before then --
+                right when the listen launches. The numeric instance ID is known only after
+                registration. For anonymous replicas, the value is ``0`` until they are officially registered.
+
+        *   -   box.status
+            -   Generic blob about the instance status. There are the most frequently used
+                and not frequently changed :doc:`config options </reference/reference_lua/box_cfg>` and
+                :doc:`box.info </reference/reference_lua/box_info>` fields.
+
+        *   -   box.election
+            -   All the required parts of :doc:`box.info.election </reference/reference_lua/box_info/election>`
+                needed to find out who is the most recent writable leader.
+
+        *   -   box.schema
+            -   Schema-related data. Currently, it contains only the version.
+
+The value for each of the built-in events is written in the following code-block:.
+
+..  code-block:: lua
+
+    -- box.id value
+    {
+    MP_STR “id”: MP_UINT; box.info.id,
+    MP_STR “instance_uuid”: MP_UUID; box.info.uuid,
+    MP_STR “replicaset_uuid”: MP_UUID box.info.cluster.uuid,
+    }
+
+    -- box.status value
+    {
+    MP_STR “is_ro”: MP_BOOL box.info.ro,
+    MP_STR “is_ro_cfg”: MP_BOOL box.cfg.read_only,
+    MP_STR “status”: MP_STR box.info.status,
+    }
+
+    -- box.election value
+    {
+    MP_STR “term”: MP_UINT box.info.election.term,
+    MP_STR “role”: MP_STR box.info.election.state,
+    MP_STR “is_ro”: MP_BOOL box.info.ro,
+    MP_STR “leader”: MP_UINT box.info.election.leader,
+    }
+
+    -- box.schema value
+    {
+    MP_STR “version”: MP_UINT schema_version,
+    }
+
+The events are available from the beginning as non-:ref:`MP_NIL <box_protocol-notation>`.
+It is necessary for supported local subscriptions.
+Otherwise, there is no way to detect whether an event is supported at all by this Tarantool version.
+If the events are broadcast before :doc:`box.cfg{} </reference/reference_lua/box_cfg>`,
+then the following values are available:
```

Comment:
It looks like an extract from a dev discussion. A doc should state facts clearly and unequivocally, without going into details why a particular design decision was made. Anyway, this paragraph is really hard for understanding.